people/methods/tag.js

```diff
@@ -0,0 +1,115 @@
+var techwords=[ "Access" ,"Account", "Android" ,"Archive" , "Artifical" ,"Application" ,"blog" ,
+    "Bit" ,"Bandwidth" ,"Bluetooth" , "Bios" ,"Broadband" ,"CAD" ,"cache" ,"CD" ,"Code" ,
+    "Cloud" ,"Compilation" ,"Connection" ,"Cookies" ,"Copyright" ,"Cyberspace","Cybersecurity",
+    "Daemon" ,"Data" ,"Database" ,"Debug" ,"Decipher" ,"DevOps" ,"digital" ,"Docker" ,
+    "Email" ,"Ethernet" , "Express" ,"encryption" ,"Facebook" ,"Fax" ,"File" ,"Film",
+    "Firewall" ,"Font" ,"Graphics" ,"Git" ,"Grid" ,"Hacker" ,"Host" ,"Hypertext" ,
+    "Internet" ,"Interface" ,"Intranet" ,"ipad" ,"job" ,"javascript" ,"js" ,"Jenkins" ,
+    "laptop" ,"LAN" ,"link" , "mainframe" ,"matrix" ,"media" ,"memory" ,"message" , "method" ,
+    "mongoDB" ,"ML" ,"machine" ,"learning" ,"MYSQL" ,"modem" ,"monitor" ,"network" ,"node" ,
+    "OS" ,"password" , "pixel" ,"php" ,"privacy" ,"project" ,"protocol" ,"quality" , "query" ,
+    "queue" ,"radio" ,"router" , "rate" ,"Technology" ,"Satellite" ,"scan" ,"Selenium" ,"Skype" ,
+    "smartphone" , "script" ,"terminal" ,"tablet" ,"tagging" ,"tracking" ,"tools" ,"Terminal" ,
+    "URL","virus" ,"video" ,"virtual" ,"virus" ,"visual" ,"Web" ,"windows" ,"Wiki" ,"word" ,
+    "Zombie",
+];
+
+var mysql = require('mysql');
+var cors = require('cors');
+var express = require('express');
+var app = express();
+app.use(cors());
+
+const con = mysql.createConnection({
+  host: "localhost",
+  user: "root",
+  password: "123456",
+  database: "interns"
+});
+
+con.connect(function (err) {
+  if (err) console.log(err);
+});
+
+var techcount=[];
+
+const sort= (techcount)=>{
```

Comment:
what does sort function returns ?  

ideally it should take (techcount and return a sorted results the function is not usable